info.xml

```diff
@@ -8,12 +8,11 @@
     <author>Ginkgo Street Labs</author>
     <email>inquire@ginkgostreet.com</email>
   </maintainer>
-  <releaseDate>2018-06-29</releaseDate>
-  <version>4.7.31-1.1.2</version>
+  <releaseDate>2020-10-13</releaseDate>
+  <version>4.7.31-1.2.0</version>
```

Comment:
The versioning convention in this extension is unique, but if we follow the pattern then this version number should be `5.23-1.2.0`